stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -419,12 +369,8 @@ extension DoubleWidth: FixedWidthInteger {
   @_inlineable // FIXME(sil-serialize-all)
   public func remainderReportingOverflow(dividingBy other: DoubleWidth)
     -> (partialValue: DoubleWidth, overflow: Bool) {
-    if other == 0 ||
-      (DoubleWidth.isSigned && other == -1 && self == .min)
-    {
-      return (self, true)
-    }
-
+    if other == (0 as DoubleWidth) { return (self, true) }
+    if DoubleWidth.isSigned && other == -1 && self == .min { return (0, true) }
```

Comment:
Can we add a test for this case?

Reply:
Yup, as mentioned, this PR is missing any tests, and I intend to cover all changes thoroughly with new tests before this is merged.

Reply:
Terrific, thank you.